recipes/dummydata/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "dummydata" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "bbd7f41df65da507bf23ff86ca5bd754c8cc9d29bf7e12a5f02b647b9273999a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py>27]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - netcdf4
+    - setuptools
+  run:
+    - python
+    - numpy x.x
```

Comment:
drop the `x.x` after numpy. You only need to include `x.x` after numpy in build and run requirements if you're linking against the numpy C API, which this package is not.